cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/opencv-control.tsx

```diff
@@ -306,21 +306,25 @@ class OpenCVControlComponent extends React.PureComponent<Props & DispatchToProps
     private disableImageModifier(alias: string):void {
         const { activeImageModifiers } = this.state;
         const index = activeImageModifiers.findIndex((imageModifier) => imageModifier.alias === alias);
+        const canvas: HTMLCanvasElement | undefined = window.document.getElementById('cvat_canvas_background');
```

Comment:
I am not sure that writing this code in this method is the correct way.
disableImageModifier can get different modifiers in general, not only equalization. Some can be disabled, some can be enabled. The same about enableImageModifier. 

I suppose it would be better to write this code in ``components/annotation-page/standard-workspace/controls-side-bar/opencv-control.tsx::renderImageContent``.

But on the other hand, adding this CSS class does not guarantee us that the equalization code works (in both places, actually). With the same level of success, we can track the class list of the equalization button. Probably better to use screenshot testing here, because we know that the gray image will be changed significantly after equaliation.

@klakhov, @dvkruchinin What do you think?

Reply:
As I said without screenshots comparison, the activated button does not guarantee that the algorithm works.

Reply:
Another option is to show a notification if something goes wrong. As I understand, we do not have such the notification now. @klakhov could help with adding it.

Reply:
I agree that its not the best way to determine that histogram was equalized. But does screenshots comparison work well enough to do it? I'm not sure but maybe it will fail from time to time.
It seems that notification or class is the safest choice.

Reply:
Generally, typical screenshot is changed significantly, I do not expect the test to be failed time to time. 

![Screenshot from 2021-08-05 15-49-47](https://user-images.githubusercontent.com/40690378/128352671-880f65fe-ee26-4795-91ff-99fb676e9c42.png)
![Screenshot from 2021-08-05 15-49-55](https://user-images.githubusercontent.com/40690378/128352674-da2d1b0d-34b3-4e41-b1b7-3a87a56577eb.png)

But if you believe it is not the best option, okay. In this case, could you add a notification? @klakhov

Reply:
Yes, I think its better to add something like this